keyrings/alt/file_base.py

```diff
@@ -61,6 +97,7 @@ def get_password(self, service, username):
         """
         service = escape_for_ini(service)
         username = escape_for_ini(username)
+        assoc = (service + '\0' + username).encode()
```

Comment:
Does it make sense for assoc to be a bytestring? Could it be a dict or tuple or just text?

The null byte here is disconcerting. It feels like a `C-style` hack. Why the null byte and not some other arbitrary character?